### PR TITLE
added tagged block to read header

### PR DESCRIPTION
**Pull request by SReich-EMI**
I added the comments of my old request. I couldn't reopen the old one and use it.

### CODE REVIEW

File: pycine/file.py [hunk 1]
```diff
@@ -55,20 +83,16 @@ def write_header(cine_file, header, backup=True):
     with open_ignoring_read_only(cine_file, "rb+") as f:
         f.write(header["cinefileheader"])
         f.write(header["bitmapinfoheader"])
-        f.seek(header["cinefileheader"].OffSetup)
```
Comment (cbenhagen) [inline pycine/file.py:58]: Why is this removed?

Comment (SReich-EMI) [reply pycine/file.py:58]: in fact I don't know, why I don't had it in my version. It seems to work like this too. 
In the file docu (version 2020) there is no gap between the BITMAPINFOHEADER and the SETUP part, so it should be fine without the seek(). But for savety reasons this can be left in.
I have added it in a new commit
